multivac/fetch.py

```diff
@@ -388,6 +389,8 @@ def download_workflow_run_jobs(workflow_run_id):
             if not job.is_stored:
                 if not args.nologs:
                     job.download_log()
+                    request_counter += 1
+                    info('Request #{}', request_counter)
```

Comment:
As long as I remember I suggested logging every 100th request, but not using a custom counter. Please log the remaining requests from `x-ratelimit-remaining` header which you have in every response from GitHub api when `x-ratelimit-remaining % 100 == 0`, every 100th.

Reply:
doc: https://docs.github.com/en/rest/overview/resources-in-the-rest-api#checking-your-rate-limit-status